Jenkinsfile

```diff
@@ -0,0 +1,78 @@
+pipeline
+{
+    agent
+    {
+        node
+        {
+            label 'embedded-docker'
+        }
+    }
+    environment
+    {
+        MAKEFLAGS = '-j 3'
+        PACKAGE_NAME = 'um_kernel'
+        DOCKER_CONTAINER_NAME = 'um-kernel_build'
+        DOCKER_IMAGE_NAME = 'um-kernel_image'
+        BUILD_OUTPUT_DIR = '_build_armhf'
```

Comment:
i'm not sure i'm happy with the name and output dir vars either. the build output dir should be named on the architecture we build for, the package name is determined outside the scope of this jenkins file